cmd/gitops/ui/run/cmd.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/weaveworks/weave-gitops/pkg/kube"
 	"github.com/weaveworks/weave-gitops/pkg/server"
 	"github.com/weaveworks/weave-gitops/pkg/server/auth"
+	wego_tls "github.com/weaveworks/weave-gitops/pkg/server/tls"
```

Comment:
You can import this as just tls (i.e. without the import alias) as it's not conflicting with anything else?